go.mod

```diff
@@ -10,13 +10,27 @@ require (
 	github.com/prometheus/common v0.37.0
 	github.com/spf13/pflag v1.0.5
 	golang.org/x/crypto v0.0.0-20220722155217-630584e8d5aa
-	k8s.io/api v0.23.9
-	k8s.io/apimachinery v0.23.9
+	k8s.io/api v0.24.0
```

Comment:
this version cause no conflict

Reply:
What was the conflict?

Reply:
couldn't build hosted mode. when it creates a container, go mod install failed at k8s.io/client-go modules.

Reply:
v 25 also error